conf/messages.en

```diff
@@ -396,7 +396,7 @@ previousEuCountry.error.duplicate = You selected this EU country already. Select
 previousOssNumber.title = Enter your registration number for {0}?
 previousOssNumber.heading = Enter your registration number for {0}?
 previousOssNumber.error.required = Enter your registration number for {0}
-previousOssNumber.error.invalid = Enter a VAT registration number in the correct format
+previousOssNumber.error.invalid = Enter registration number in the correct format
```

Comment:
Is this missing an "a"? Enter a registration number in the correct format

Reply:
Added missing character 'a'